components/automate-ui/src/styles.scss

```diff
@@ -17,6 +17,12 @@
   }
 }
 
+.sticky-notifications {
+  position: sticky;
+  top: 0;
+  z-index: 199;
+}
+
```

Comment:
This keeps all notifications _except_ the license notification pinned to the top on scroll.